README.md

```diff
@@ -51,3 +53,30 @@ npm run test
 ```plain
 npm run doc
 ```
+## Usage
+After installing *As*, you can import the object classes and functions that you need in your AssemblyScript file.
+
+For example, to use the `combination` function to determine the combination of 2 numbers, you can import and use it like this:
+```typescript
+import { combination } from  '@massalabs/as-proba';
+import { Args } from  '@massalabs/as-types';
+
+// This main function is called automatically when the smart contract is executed by the blockchain.
+//the argument args contains the serialized values n and k (n >= k) as U64
+export function main(args: StaticArray<u8>): void {
+	const  n = args.nextU64();
+	const  k = args.nextU64();
+	const  result = combination(n, k);
+	generateEvent(`The result of the combination of ${n.toString()} and ${k.toString()} is ${result.toString()}`);
+}
+```
+## Contributing
+We welcome contributions from the community!
+
+If you would like to contribute to Massa-as-sdk, please read the [CONTRIBUTING file](CONTRIBUTING.md).
+
+## License
+Massa-as-sdk is released under the [MIT License](LICENSE).
+
+## Powered By
+Massa-as-sdk is developed with love by MassaLabs and powered by a variety of [open-source projects](powered-by.md).
```

Comment:
Bad copy/paste ?

Reply:
We havent created this files yet. Once we do it it will work

Reply:
Not talking about the link the the fact that you are talking about Massa-as-sdk